docs/methoddocs/Detectionlists.md

```diff
@@ -0,0 +1,25 @@
+# Detection Lists
```

Comment:
This file name still starts with a capital letter, causing the docs not to build

Reply:
wat, i just changed this howwwwww

Reply:
It is because of GitHub
it's annoying

Reply:
I think the only way to fix it is to first delete it. Commit. then recreate it

Reply:
you can rename it in the browser

Reply:
TIL